app/components/Header.test.tsx

```diff
@@ -0,0 +1,29 @@
+import '@testing-library/jest-dom'
+import { fireEvent, render, screen, waitFor } from '@testing-library/react'
```

Comment:
although fireEvent is not incorrect. userEvent is a better way of testing user events.

you can see how it's implemented on another project of mine at: 

https://github.com/bgwebagency/redesigner.io/blob/f5794c29d77ae7b5384aebc6f02d248c4fce7edf/app/components/Footer/__test__/Footer.test.tsx#L4

Read more at: https://testing-library.com/docs/user-event/intro/#differences-from-fireevent

Reply:
Did it!

Reply:
Done!